src/io/flutter/run/test/FlutterTestLineMarkerContributor.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.run.test;
+
+import com.intellij.execution.lineMarker.ExecutorAction;
+import com.intellij.execution.lineMarker.RunLineMarkerContributor;
+import com.intellij.icons.AllIcons;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.util.text.StringUtil;
+import com.intellij.psi.PsiElement;
+import com.intellij.util.Function;
+import com.intellij.util.containers.ContainerUtil;
+import com.jetbrains.lang.dart.psi.DartCallExpression;
+import io.flutter.dart.DartSyntax;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Objects;
+
+
+public class FlutterTestLineMarkerContributor extends RunLineMarkerContributor {
+  @Nullable
+  @Override
+  public Info getInfo(@NotNull PsiElement element) {
+    if (isTestCall(element)) {
+      final AnAction[] actions = ExecutorAction.getActions();
+      final Function<PsiElement, String> tooltipProvider =
+        psiElement -> StringUtil.join(ContainerUtil.mapNotNull(actions, action -> getText(action, element)), "\n");
+      return new Info(AllIcons.RunConfigurations.TestState.Run, tooltipProvider, actions);
```

Comment:
Any idea what that signifies?

Here's what I see for a JUnit test:

![image](https://user-images.githubusercontent.com/67586/27927244-fd26c7b6-623f-11e7-9b11-b897735d5b7d.png)